apps/platform/messages/en.json

```
@@ -89,5 +89,27 @@
   "languages" : {
     "en": "English",
     "hr": "Croatian"
+  },
+  "projects" : {
+    "title": "Projects",
+    "map": "Map",
+    "board": "Board",
+    "neighborhood": "By neighborhood",
+    "filterNeighborhood": "Filter by Neigbourghood",
+    "projectsMap": "Projects on the map"
+  },
+  "projectCard" : {
+    "funded": "funded"
+  },
+  "project" : {
+    "funded": "funded of €{amount} goal",
+    "bakers": "bakers",
```

Typo, unless we're baking 🥐 😆 

```suggestion
    "backers": "backers",
```

---

resolved